routes/ingredient.js

```diff
@@ -8,7 +8,9 @@ const { convertToGrams } = require('../services/weightConversions');
 //CREATE NEW INGREDIENT
 const createIngredient = (request, response) => {
     const { ingredient_name, recipe_id, product_id, ingredient_weight, ingredient_weight_type } = request.body;
-    const ingredient_gram_weight = convertToGrams(ingredient_weight, ingredient_weight_type);
+    const lowercasedType = ingredient_weight_type.toLowerCase()
```

Comment:
would recommend this variable to be named `lowercasedWeightType` since `type` isn't descriptive

Reply:
actually probably a better name is `lowercased_weight_type` since all the other variables are in snakecase.